api/python/requirements.txt

```diff
@@ -3,3 +3,4 @@ ninja
 pyyaml
 prettytable
 nose
+pytest
```

Comment:
Why this requirement?

Reply:
Without this requirement, I'm getting the following error when trying to run the Python API tests locally as shown below (it's entirely possible that I'm just not invoking the tests correctly):

```
simon@phoenix:~ $ cd ~/src/rapidyaml/api/python
simon@phoenix:~/src/rapidyaml/api/python $ make test
[ ... ]
Successfully installed rapidyaml-0.2.2.post16
Installed version: 0.2.2.post16
make[1]: Leaving directory '/home/simon/src/rapidyaml/api/python'
[[ -e venv/bin/activate ]] && source venv/bin/activate; pytest
/bin/bash: pytest: command not found
make: *** [Makefile:92: test] Error 127
simon@phoenix:~/src/rapidyaml/api/python $ 
```

Reply:
After looking at the GitHub workflows, I think the better option (to keep parity between CI and the Makefile) would be to apply the following change to `api/python/Makefile` instead of adding `pytest` to the requirements:

```diff
diff --git a/api/python/Makefile b/api/python/Makefile
index f5100b8..53db086 100644
--- a/api/python/Makefile
+++ b/api/python/Makefile
@@ -89,7 +89,7 @@ install: | $(ACTIVATE_SCRIPT)
 .PHONY: install
 
 test: | $(ACTIVATE_SCRIPT)
-       ${ACTIVATE} pytest
+       ${ACTIVATE} python -m nose ryml/tests
 
 .PHONY: test
```

Reply:
Agreed. Thanks for investigating.

Reply:
Pushed the suggested change instead of the additional dependency in the latest version